tools/online_editor/src/editor_widget.ts

```diff
@@ -476,26 +489,27 @@ class EditorPaneWidget extends Widget {
     this.#onModelSelected = f;
   }
 
-  protected async fetch_url_content(era: number, url: string): Promise<string> {
-    let model_and_state = this.#editor_documents.get(url);
-    if (model_and_state != null) {
-      return model_and_state.model.getValue();
+  protected async fetch_url_content(era: number, uri: string): Promise<string> {
+    const uri_ = monaco.Uri.parse(uri);
```

Comment:
Ah, Salme situation here: maybe use the uri type again? Or is there a problem converting it back to a string? (For use with fetch())